compiler/next/Makefile.help

```diff
@@ -71,6 +71,12 @@ libchplcomp-docs: $(LIBCOMPILER_BUILD_DIR) FORCE
 	  $(COPY_IF_DIFFERENT) $(LIBCOMPILER_BUILD_DIR)/doc/doxygen $(CHPL_MAKE_HOME)/build/doc/doxygen ; \
 	fi
 
+clean-libchplcomp-docs: FORCE
+	rm -rf $(CHPL_MAKE_HOME)/build/doc/doxygen
+
+clobber-libchplcomp: FORCE
```

Comment:
Hasn't that led to complaints in the past about one or the other not fully restoring us to a fresh state?

Reply:
At one point in history, the clobber rule was less different (and less about just removing the whole `build` directory), which yes, did cause problems. Anyway now that I look at the diff, I see what you mean, because `$(CHPL_MAKE_HOME)/build/doc/doxygen` is not a subdir of `$(LIBCOMPILER_BUILD_DIR)` (and in the discussion, I was misremembering that). I'll make clobber-libchplcomp also call clean-libchplcomp-docs.